### DISCUSSION TITLE
im getting error in downloading foundry on my pc

### DISCUSSION THREAD

**Discussion by TaiwoEniola** [Cyfrin/foundry-full-course-cu#2640] [General]
i keep getting error for the chisel, and the other three

**Comment by Shiva-Sai-ssb** [comment]
What error are you getting? @TaiwoEniola

**Comment by TaiwoEniola** [comment]
this is the error im getting :
root@DESKTOP-QPB1LHG:/mnt/c/Users/ADMIN# curl -L https://foundry.paradigm.xyz | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed  
100   167  100   167    0     0     51      0  0:00:03  0:00:03 --:--:--    51 
100  2189  100  2189    0     0    206      0  0:00:10  0:00:10 --:--:--   497
Installing foundryup...

Detected your preferred shell is bash and added foundryup to PATH.
Run 'source /root/.bashrc' or start a new terminal session to use foundryup.
Then, simply run 'foundryup' to install Foundry.
root@DESKTOP-QPB1LHG:/mnt/c/Users/ADMIN# foundryup


.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx

 ╔═╗ ╔═╗ ╦ ╦ ╔╗╔ ╔╦╗ ╦═╗ ╦ ╦         Portable and modular toolkit
 ╠╣  ║ ║ ║ ║ ║║║  ║║ ╠╦╝ ╚╦╝    for Ethereum Application Development
 ╚   ╚═╝ ╚═╝ ╝╚╝ ═╩╝ ╩╚═  ╩                 written in Rust.

.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx

Repo       : https://github.com/foundry-rs/
Book       : https://book.getfoundry.sh/
Chat       : https://t.me/foundry_rs/
Support    : https://t.me/foundry_support/
Contribute : https://github.com/orgs/foundry-rs/projects/2/

.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx

foundryup: installing foundry (version nightly, tag nightly)
foundryup: downloading latest forge, cast, anvil, and chisel
                    tar:   Child died with signal 13                                                  0.0%  tar:  Error is not recoverable: exiting now
                                                                          0.0%foundryup: command failed: tar -xzC /root/.foundry/bin
curl: (23) Failure writing output to destination

foundryup: command failed: download https://github.com/foundry-rs/foundry/releases/download/nightly/foundry_nightly_linux_amd64.tar.gz
root@DESKTOP-QPB1LHG:/mnt/c/Users/ADMIN#

**Reply by WhatMe1on** [reply]
After run 'foundryup' command for several times ,my foundry just works, I guess is the Network fluctuation cause
```
foundryup: command failed: download https://github.com/foundry-rs/foundry/releases/download/nightly/foundry_nightly_linux_amd64.tar.gz
```
and caused this series of problems

**Reply by EngrPips** [reply]
Unstable network is a great problem

**Comment by TaiwoEniola** [comment]
this is the error im getting :
root@DESKTOP-QPB1LHG:/mnt/c/Users/ADMIN# curl -L https://foundry.paradigm.xyz | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed  
100   167  100   167    0     0     51      0  0:00:03  0:00:03 --:--:--    51 
100  2189  100  2189    0     0    206      0  0:00:10  0:00:10 --:--:--   497
Installing foundryup...

Detected your preferred shell is bash and added foundryup to PATH.
Run 'source /root/.bashrc' or start a new terminal session to use foundryup.
Then, simply run 'foundryup' to install Foundry.
root@DESKTOP-QPB1LHG:/mnt/c/Users/ADMIN# foundryup


.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx

 ╔═╗ ╔═╗ ╦ ╦ ╔╗╔ ╔╦╗ ╦═╗ ╦ ╦         Portable and modular toolkit
 ╠╣  ║ ║ ║ ║ ║║║  ║║ ╠╦╝ ╚╦╝    for Ethereum Application Development
 ╚   ╚═╝ ╚═╝ ╝╚╝ ═╩╝ ╩╚═  ╩                 written in Rust.

.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx

Repo       : https://github.com/foundry-rs/
Book       : https://book.getfoundry.sh/
Chat       : https://t.me/foundry_rs/
Support    : https://t.me/foundry_support/
Contribute : https://github.com/orgs/foundry-rs/projects/2/

.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx

foundryup: installing foundry (version nightly, tag nightly)
foundryup: downloading latest forge, cast, anvil, and chisel
                    tar:   Child died with signal 13                                                  0.0%  tar:  Error is not recoverable: exiting now
                                                                          0.0%foundryup: command failed: tar -xzC /root/.foundry/bin
curl: (23) Failure writing output to destination

foundryup: command failed: download https://github.com/foundry-rs/foundry/releases/download/nightly/foundry_nightly_linux_amd64.tar.gz
root@DESKTOP-QPB1LHG:/mnt/c/Users/ADMIN#

**Comment by TaiwoEniola** [comment]
this is the error im getting :
root@DESKTOP-QPB1LHG:/mnt/c/Users/ADMIN# curl -L https://foundry.paradigm.xyz | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed  
100   167  100   167    0     0     51      0  0:00:03  0:00:03 --:--:--    51 
100  2189  100  2189    0     0    206      0  0:00:10  0:00:10 --:--:--   497
Installing foundryup...

Detected your preferred shell is bash and added foundryup to PATH.
Run 'source /root/.bashrc' or start a new terminal session to use foundryup.
Then, simply run 'foundryup' to install Foundry.
root@DESKTOP-QPB1LHG:/mnt/c/Users/ADMIN# foundryup


.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx

 ╔═╗ ╔═╗ ╦ ╦ ╔╗╔ ╔╦╗ ╦═╗ ╦ ╦         Portable and modular toolkit
 ╠╣  ║ ║ ║ ║ ║║║  ║║ ╠╦╝ ╚╦╝    for Ethereum Application Development
 ╚   ╚═╝ ╚═╝ ╝╚╝ ═╩╝ ╩╚═  ╩                 written in Rust.

.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx

Repo       : https://github.com/foundry-rs/
Book       : https://book.getfoundry.sh/
Chat       : https://t.me/foundry_rs/
Support    : https://t.me/foundry_support/
Contribute : https://github.com/orgs/foundry-rs/projects/2/

.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx.xOx

foundryup: installing foundry (version nightly, tag nightly)
foundryup: downloading latest forge, cast, anvil, and chisel
                    tar:   Child died with signal 13                                                  0.0%  tar:  Error is not recoverable: exiting now
                                                                          0.0%foundryup: command failed: tar -xzC /root/.foundry/bin
curl: (23) Failure writing output to destination

foundryup: command failed: download https://github.com/foundry-rs/foundry/releases/download/nightly/foundry_nightly_linux_amd64.tar.gz
root@DESKTOP-QPB1LHG:/mnt/c/Users/ADMIN#

**Reply by EngrPips** [reply]
Did you follow this instruction `Run 'source /root/.bashrc' or start a new terminal session to use foundryup.`

**Reply by TaiwoEniola** [reply]
yes i did

**Reply by TaiwoEniola** [reply]
its still the same thing im getting

**Reply by EngrPips** [reply]
Hello @TaiwoEniola, Please read this StackOverflow post https://stackoverflow.com/questions/3950839/tar-error-is-not-recoverable-exiting-now and let's see if it helps you.
